thundra/opentracing/tracer.py

```diff
@@ -24,6 +24,7 @@ def __init__(self, recorder=None, scope_manager=None):
         self.global_span_order = 0
         self.test_xray_traces = []
```

Comment:
there should not be `test_xray_traces` here? I hope that this is redundant or left over